docs/acceptance-guide.mdx

```diff
@@ -220,14 +194,80 @@ https://acme-bank.co/sign-up/complete?sharedCredentialsUuid=37ce0fff-af32-457e-9
 
 It's easiest to extract the data from each credential using the corresponding [schema](/schema). The extracted data is verified according to the issuer of the credential, but you can perform additional verification steps if you like.
 
+<CredentialUsageExample/>
+
 ✅ **_You're done!_**
 
-### Partner Acceptance
+### General Acceptance
+
+Because the issuer is not issuing credentials with your brand in mind and handling the redirect to the Verified Inc. Wallet, **you need to check whether the user has matching <Tip type="credential">credentials</Tip>.** If they do, they can share all of those credentials with 1-click.
```

Comment:
```suggestion
Because the issuer is not issuing credentials with your brand in mind, **you need to check whether the user has matching <Tip type="credential">credentials</Tip>.** If they do, they can share all of those credentials with 1-click. **You will also need to direct the user to the Verified Inc. Wallet.**
```